Makefile

```diff
@@ -1,22 +1,28 @@
-PROMBENCH_CMD        = /bin/prombench
+PROMBENCH_CMD        = ./prombench
 DOCKER_TAG = docker.io/sipian/prombench:v2.0.0
 
+#Prow config has the following args set in it's configuration during deployment
+#	ZONE
+#	PROJECT_ID
+#	CLUSTER_NAME
+#
+#When the start-benchmark/cancel-benchmark prow-job is created, the benchmark plugin adds the following args
+#	PR_NUMBER
+#	RELEASE - which release version to benchmark PR with (Default: master)
```

Comment:
Why do you think we need these comment?

This should be  just s simple Dockerimage which expects these variables.
I think we should try to separate it as much as possible from any other notions. Otherwise need to constantly update comments when we change the logic somewhere else.

Reply:
i have removed the entire comment